app/models/test_set.rb

```diff
@@ -12,6 +12,8 @@ def for_task(task)
 
   validates :name, presence: true
 
+  scope :published, -> { where(published: true) }
```

Comment:
Not sure I'm following here. Can you show some examples of the problem you are describing?

Reply:
Can we do something like this from the UI? I believe this can stay as it is right now. Also just to reference, see this thread: https://x.com/dhh/status/1864037874485862508

Reply:
I see your point. After thinking about this deeper, it seems like this value can be nil, and then it is treated as `false` - which is the default. I modified the migration and remove `null: false`.